main.go

```diff
@@ -223,6 +224,16 @@ func main() { // nolint:gocyclo
 		setupLog.Error(err, "unable to create controller", "controller", "ToolchainClusterCache")
 		os.Exit(1)
 	}
+
+	if err := (&toolchaincluster.Reconciler{
+		Client:     mgr.GetClient(),
+		Scheme:     mgr.GetScheme(),
+		RequeAfter: 10 * time.Second,
+	}).SetupWithManager(mgr); err != nil {
+		setupLog.Error(err, "unable to create controller", "controller", "ToolchainClusterHealth")
```

Comment:
```suggestion
		setupLog.Error(err, "unable to create controller", "controller", "ToolchainCluster")
```
I still see it as 'ToolchainClusterHealth' in the PR, maybe the change got reverted somehow?

Reply:
done